search-app/src/cmr/search/results_handlers/stac_results_handler.clj

```diff
@@ -218,3 +223,63 @@
 (defmethod qs/single-result->response [:granule :stac]
   [context query results]
   (single-result->response context query results))
+
+(defn- get-fc-links
+  "Returns the links for feature collection"
+  [context query hits]
+  (let [{:keys [page-size offset]} query
+        page-num (if offset
+                   (inc (/ offset page-size))
```

Comment:
Would `(inc (quot offset page-size))` be safer?

Reply:
the `offset` here is the calculated result of page-size * page-num, so there is no need to worry about what if it does not fully divid.